geniza/corpus/admin.py

```diff
@@ -197,7 +218,16 @@ def get_queryset(self, request):
         return (
             super()
             .get_queryset(request)
-            .prefetch_related("tags", "languages", "textblock_set")
+            .select_related("doctype")
+            .prefetch_related(
+                "tags",
+                "languages",
+                Prefetch(
+                    "textblock_set",
+                    queryset=TextBlock.objects.select_related("fragment"),
+                ),
+                "footnotes__content__isnull",
```

Comment:
Can you add a comment documenting this? I know you explained it to me but I already forgot